config.json

```diff
@@ -505,995 +339,1478 @@
         "uuid": "29c3c9ff-c7f3-4646-9612-7cd768d237ab",
-        "prerequisites": [],
+        "prerequisites": [
+          "cond",
+          "case",
+          "if",
+          "multiple-clause-functions",
+          "pattern-matching",
+          "binaries",
+          "recursion",
+          "tail-call-recursion",
+          "enum",
+          "strings"
+        ],
+        "practices": ["recursion", "tail-call-recursion", "strings"],
         "difficulty": 3,
-        "topics": [
-          "binary_operators",
-          "math"
-        ]
+        "topics": ["binary_operators", "math"]
       },
       {
         "slug": "change",
         "name": "Change",
         "uuid": "14bddd31-a68f-4a16-830f-6e8ecf8199f2",
-        "prerequisites": [],
-        "difficulty": 3,
-        "topics": [
-          "enumeration",
-          "reduce"
-        ]
+        "prerequisites": [
+          "integers",
+          "atoms",
+          "tuples",
+          "multiple-clause-functions",
+          "pattern-matching",
+          "guards",
+          "recursion",
+          "enum",
+          "maps"
+        ],
+        "practices": ["integers", "enum"],
+        "difficulty": 7,
+        "topics": ["enumeration", "reduce"]
       },
       {
         "slug": "transpose",
         "name": "Transpose",
         "uuid": "ce270a34-add1-422c-bb86-53b310f05e27",
-        "prerequisites": [],
+        "prerequisites": ["strings", "enum"],
+        "practices": ["strings", "enum"],
         "difficulty": 5,
-        "topics": [
-          "strings",
-          "transforming"
-        ]
+        "topics": ["strings", "transforming"]
       },
       {
         "slug": "ocr-numbers",
         "name": "OCR Numbers",
         "uuid": "3f321191-afb1-4aa1-995b-204aa460269e",
-        "prerequisites": [],
+        "prerequisites": [
+          "strings",
+          "charlists",
+          "multiple-clause-functions",
+          "pattern-matching",
+          "cond",
+          "case",
+          "if",
+          "enum",
+          "maps",
+          "binaries"
+        ],
+        "practices": [
+          "strings",
+          "multiple-clause-functions",
+          "pattern-matching",
+          "enum"
+        ],
         "difficulty": 6,
-        "topics": [
-          "enumerations",
-          "strings"
-        ]
+        "topics": ["enumerations", "strings"]
       },
       {
         "slug": "kindergarten-garden",
         "name": "Kindergarten Garden",
         "uuid": "ae72ff68-e4c9-472c-8a86-7adaaa9086ee",
-        "prerequisites": [],
+        "prerequisites": [
+          "tuples",
+          "lists",
+          "string",
+          "multiple-clause-functions",
+          "pattern-matching",
+          "default-arguments",
+          "recursion",
+          "enum",
+          "maps",
+          "ranges",
+          "regular-expressions",
+          "module-attributes-as-constants"
+        ],
+        "practices": [
+          "string",
```

Comment:
Possible typo here (and in the prerequisites of this exercise): there is no concept `string`, but there is `strings` (plural).

Reply:
```suggestion
          "strings",
```

Reply:
Good eye 👁️. There was one more typo like that. I did an automated check and there are no more typos 🎉